### PR TITLE
Review spinner component

### PR DESCRIPTION
**Pull request by tioluwani94**
This PR includes updates made to the spinner component package in preparation for release. The following activities were carried out:

- Tests review
- Types review
- Storybook examples
- Comments review
- Readme review
- Dependency list review

### CODE REVIEW

File: packages/spinner/README.md [hunk 1]
```diff
@@ -1 +1,66 @@
...
+Change the background color of the moving section of the spinner by passing the
+`color` prop.
+
+```jsx
+<Spinner color="red.500" />
+```
+
+## Spinner with empty area color
+
+Change the background color of the spinner's by passing the `emptyColor` prop.
```
Comment (with-heart) [inline packages/spinner/README.md:53]:
```suggestion
Change the background color of the spinner by passing the `emptyColor` prop.
```

File: packages/spinner/README.md [hunk 2]
```diff
@@ -1 +1,66 @@
...
+
+```jsx
+<Spinner color="red.500" emptyColor="gray.200" />
+```
+
+## Spinner with different speed
+
+Change the spinner's animation speed area by passing the `speed` prop. The unit
+of the value is seconds
```
Comment (with-heart) [inline packages/spinner/README.md:62]:
```suggestion
of the value is seconds.
```

Comment (with-heart) [inline packages/spinner/README.md:62]: We could also reference `animation-duration` instead:

```suggestion
Change the spinner's animation speed area by passing the `speed` prop. The unit
of the value matches the [`animation-duration CSS property`](https://developer.mozilla.org/en-US/docs/Web/CSS/animation-duration) property.
```

File: packages/spinner/README.md (hunk 1 above)
Comment (tioluwani94) [reply packages/spinner/README.md:53]: Thanks for catching this @with-heart.

File: packages/spinner/README.md (hunk 2 above)
Comment (tioluwani94) [reply packages/spinner/README.md:62]: Thanks for catching this @with-heart.

Comment (tioluwani94) [reply packages/spinner/README.md:62]: Thanks for this @with-heart.

**Pull request merged**